frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -553,6 +554,11 @@ export default ({
         this.updateMentionKeyword()
       }
     },
+    handlePaste (e) {
+      const pastedText = e.clipboardData.getData('text')
+      this.$refs.textarea.value = pastedText
```

Comment:
This will replace all existing text with the pasted text, overwriting the value.

Instead of that, we want the pasted value to be inserted where the cursor is like normal.

Reply:
My suggestion of using the `input` event doesn't need this `handlePaste` handler (it needs a much simpler handler, as shown in my example). However, I didn't thoroughly test it.

Reply:
It does not make sense to add `input` when there is `keyodwn` already in place. It has to be either one of them. They are two events that look similar but behave differently in certain cases in my experience. For that reason it's a type of change I would be very cautious to do at this point (Considering this code has been this way for a long while). Also, as I said above, I think other front-end devs chose to use `keydown` instead of `input` for a reason.